kafka-connector/src/main/java/com/google/pubsub/kafka/source/CloudPubSubSourceTask.java

```diff
@@ -59,18 +61,16 @@
   private String cpsSubscription;
   private String kafkaMessageKeyAttribute;
   private String kafkaMessageTimestampAttribute;
-  private int kafkaPartitions;
   private PartitionScheme kafkaPartitionScheme;
   private int cpsMaxBatchSize;
-  // Keeps track of the current partition to publish to if the partition scheme is round robin.
-  private int currentRoundRobinPartition = -1;
   // Keep track of all ack ids that have not been sent correctly acked yet.
   private Set<String> ackIds = Collections.synchronizedSet(new HashSet<String>());
   private CloudPubSubSubscriber subscriber;
   private Set<String> ackIdsInFlight = Collections.synchronizedSet(new HashSet<String>());
   private final Set<String> standardAttributes = new HashSet<>();
 
-  public CloudPubSubSourceTask() {}
+  public CloudPubSubSourceTask() {
```

Comment:
Can you please undo all of these formatting changes? I'd rather they not be included with a functional change so it can be clear what is relevant to the change and what isn't.